src/main.rs

```diff
@@ -595,6 +597,7 @@ fn main() {
       )
       (@arg DETACH: -d "detach projects back to manifest revision")
       (@arg REFS_ONLY: -r --("refs-only") "don't checkout, only update the refs")
+      (@arg PULL_LFS: -f --lfs "pull git LFS for all the supported repos")
```

Comment:
I don't think this deserves a short flag (especially not `-f`), since it should probably just be automatic (with `--no-lfs`, and global/tree options to disable it).